hbci_client.py

```diff
@@ -52,46 +96,25 @@ def get_connection(ask_for_tan=terminal_tan_callback):
     if not pin:
         pin = getpass.getpass("Please enter PIN: ")
 
-    data = None
-    try:
-        with open("data/fints_state.bin", "rb") as fd:
-            data = fd.read()
-    except IOError as e:
-        logging.info("State not recovered: %s", e)
-
-    conn = FinTS3PinTanClient(
-        bank_identifier=config.blz,
-        user_id=config.user,
-        pin=pin,
-        server=config.fints_url,
-        product_id=config.product_id,
-        mode=FinTSClientMode.INTERACTIVE,
-        product_version=version,
-        from_data=data
-    )
-    #conn._need_twostep_tan_for_segment = lambda _: True
-    #conn.fetch_tan_mechanisms()
-    #conn.set_tan_mechanism('910')
-    #conn.set_tan_mechanism('942')   # eg for mobiletan at gls
-
-    if conn.init_tan_response:
-        logging.info("Calling tan callback for init_tan_response %s", conn.init_tan_response)
-        tan = ask_for_tan(conn.init_tan_response)
-        conn.send_tan(conn.init_tan_response, tan)
-    
+    # TODO: ask_for_tan needs to be passed through, so that it
+    #       also works in the web interface
+    # conn = client_from_config(ask_for_tan)
```

Comment:
TODO marker

Reply:
Discussed with @rfl-root, we would like to make it work in the django admin interface, but focus for now is to get the project running again on the server. So skipping for now, even if it means that entering the tan is only possible via CLI.